cupyx/scipy/spatial/distance.py

```diff
@@ -114,7 +114,7 @@ def __init__(self, canonical_name=None, aka=None,
 def check_soft_dependencies():
     if not cuvs_available:
         if not pylibraft_available:
-            raise RuntimeError('cuVS >= 24.12 or pylibraft < '
+            raise RuntimeError('cuVS >= 25.02 or pylibraft < '
```

Comment:
Went ahead and did this along with CI changes in PR: https://github.com/cupy/cupy/pull/8970

Let's go ahead and drop this from here

```suggestion
            raise RuntimeError('cuVS >= 24.12 or pylibraft < '
```